src/graph/backend/dnnl/thread_local_cache.hpp

```diff
@@ -171,7 +171,7 @@ class thread_local_cache_t {
 private:
     class global_cache_type_t {
     public:
-        global_cache_type_t() : mutex_(), data_(), counter_(1) {}
+        global_cache_type_t() : data_(), counter_(1) {}
```

Comment:
I fixed this (the clang-tidy warning IIUC) on main branch. Could you please rebase? Thanks!

Reply:
Rebased, thank you for taking care of it!